elastic/kibana
# [OAS] Support tags

## Summary

Close https://github.com/elastic/kibana/issues/183935
Close https://github.com/elastic/kibana/issues/182649

## Test

1. Start Kibana + ES
2. Add a tag like `oas-tag:foo` to a route under `options.tags`
3. Generate and view OAS using command like `curl -suelastic:changeme http://localhost:5601/api/oas\?pathStartsWith\=/api/status | jq`

Should contain (note addition of ["global" tags object](https://docs.elastic.dev/content-architecture/oas#tags)):

```json
  "paths": {
    "/api/status": {
      "get": {
        "summary": "Get Kibana's current status.",
        "tags": [
          "foo"
        ],
  ...
  "tags": [{ "name": "foo" }],
```
